admin/i18n/de/translations.json

```diff
@@ -112,4 +112,6 @@
 	"lightcontrol adapter settings": "Adaptereinstellungen für LightControl",
 	"option1": "Option1",
 	"option2": "Option2"
+	"Min Color temperature in Kelvin": "Minwert Farbtemperatur in Kelvin",
```

Comment:
Übersetzungen ausschließlich in unter /en/translations.json eintragen!
Verwende bitte Color-Temperature anstatt Color temperature

Reply:
fixed